README.md

```diff
@@ -232,15 +232,15 @@ There are multiple terms that might be confused when logging a user out.
 ### Using a Credential to determine user authentication status
 There are a few options to determine the status of a user authentication. Each option has unique pros and cons and should be chosen based on the needs of your use case.
 
-- Non null default credential: `Credential.getDefaultCredential() != null`
+- Non null default credential: `Credential.getDefaultAsync() != null`
```

Comment:
I think, for consistency, we should reference the sync versions, ala `Credential.default`, etc.  It makes the code more readable, and during the natural course of the SDK, the async version is unnecessary (e.g. it doesn't go to storage with every single call, since the credential instance is cached in-memory)

Reply:
Good point. I will change this to Credential.default